apiary-ranger-metastore-plugin/src/main/java/com/expediagroup/apiary/extensions/rangerauth/listener/ApiaryRangerAuthPreEventListener.java

```diff
@@ -15,9 +15,7 @@
  */
 package com.expediagroup.apiary.extensions.rangerauth.listener;
 
-import java.util.Date;
-import java.util.Set;
-
+import com.google.common.collect.Sets;
```

Comment:
Nothing has changed here other then the import order so maybe just revert this file? We can discuss offline about the ordering rules.